nvflare/fuel/hci/zip_utils.py

```diff
@@ -13,13 +13,17 @@
 # limitations under the License.
 
 import io
+import json
 import os
 from zipfile import ZipFile
 
+from nvflare.apis.job_def import JobMetaKey
+
 META_NAME = "meta.json"
 # A format string for the dummy meta.json
 META_DATA = """{{
   "name": "{}",
+  "job_folder_name": "{}",
```

Comment:
note that these values might change `name, job_folder_name, resource_spec, deploy_map`
we could improve them in another PR to use constants.

Reply:
let me change them now